clarity.d.ts

```diff
@@ -384,7 +394,8 @@ declare const enum Instrumentation {
   ClarityDuplicated,
   ShadowDomInconsistent,
   ClarityActivateError,
-  Trigger
+  Trigger,
+  PerformanceStateError
```

Comment:
>PerformanceStateError [](start = 2, length = 21)

What does this mean? not obvious from name.

Reply:
This is the case when we detect that w3c performance entries buffer is full or has been cleared. I know it's not too obvious, but it's not a new feature and isn't a focus of this PR - i'll clean this up separately.

---
In reply to: [171736127](https://github.com/Microsoft/clarity-js/pull/70#discussion_r171736127) [](ancestors = 171736127)

Reply:
Also, there is a comment at the top of performance.ts explaining the issue

---
In reply to: [171739106](https://github.com/Microsoft/clarity-js/pull/70#discussion_r171739106) [](ancestors = 171739106,171736127)

Reply:
Can you just add a brief comment here or refer to the appropriate location as a comment here.; 

---
In reply to: [171740258](https://github.com/Microsoft/clarity-js/pull/70#discussion_r171740258) [](ancestors = 171740258,171739106,171736127)